charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -11,6 +11,12 @@ image:
 customLabels: {}
 # k8s-app: aws-ebs-csi-driver
 
+# Instruct the AWS SDK to use AWS FIPS endpoints, and deploy container built with BoringCrypto (a FIPS-validated cryptographic library) instead of the Go default
+#
+# The EBS CSI Driver FIPS images have not undergone FIPS certification, and no official guarnatee is made about the compliance of these images under the FIPS standard
+# Users relying on these images for FIPS compliance should perform their own independent evaluation
+fips: false
```

Comment:
nit: can we change it to enableFips

Reply:
I thought about this, but it's not really "enabling" FIPS so the name would be misleading. Maybe `fipsMode` or `enableFipsMode` although those sound kinda janky.